src/chrome/content/rules/Zendesk.com.xml

```diff
@@ -45,10 +46,19 @@
 	<target host="zendesk.com" />
 	<target host="*.zendesk.com" />
 
-		<exclusion pattern="^http://video\.zendesk\.com/" />
+		<exclusion pattern="^http://hello\.zendesk\.com/" />
+		<test url="http://hello.zendesk.com/" />
+
+		<!-- See https://github.com/EFForg/https-everywhere/issues/9159 -->
+		<exclusion pattern="^http://app\.hello\.zendesk\.com/" />
+		<test url="http://app.hello.zendesk.com/" />
+
+		<exclusion pattern="^http://radar\.zendesk\.com/" />
+		<test url="http://radar.zendesk.com/" />
 
+		<exclusion pattern="^http://video\.zendesk\.com/" />
```

Comment:
It was already excluded at https://github.com/EFForg/https-everywhere/pull/9161/files#diff-256722ade31af854289b496e13e802b3L48.